members.Rmd

```diff
@@ -51,61 +56,57 @@ I am interested in all kinds of statistical methods, but currently my research a
+I am a fourth-year Ph.D. quantitative psychology student, working under the supervision of Dr. Hok Chio (Mark) Lai. Broadly speaking, my research interests are in multilevel modeling, structural equation modeling, and Bayesian statistics. My main project focuses on incorporating uncertainty in parameter values (e.g., effect size and intraclass correlation) for designing multilevel studies. My secondary project investigates when strict factorial invariance is a necessary condition for valid use of observed scale scores. I am also involved in projects on latent class/profile analysis, two-stage path analysis with adjustment for measurement noninvariance and unreliability, and multilevel regression and poststratification on subnational predictions with language data.
 
-- [CV](static/cv/WinnieTse_CV_20220222.pdf)
+- [CV](https://winniewytse.rbind.io/cv/winniewytse_cv.pdf)
 - [Personal website](https://winniewytse.rbind.io/)
 - Email: wingyeet@usc.edu
 
 ### Yichi Zhang
 
 ![](static/images/YichiZhang_image_20210326.jpg){width=50%}
 
-Hello! I am a third-year Ph.D. student in quantitative psychology. I am co-advised by Dr. Hok Chio (Mark) Lai and Dr. Rand Wilcox. My current research interests are measurement invariance, robust structural equation modeling and longitudinal growth modeling. My current project is about establishing measurement invariance using a Bayesian ROME (region of measurement equivalence) method.
+I am a fourth-year Ph.D. student in Quantitative Psychology. I am co-advised by Dr. Hok Chio(Mark) Lai and Dr. Rand Wilcox. My current research interests are measurement bias in surveys/questionnaires and robust statistical models, such as hierarchical linear model(linear mixed effects model), latent variable model (structural equation modeling), etc. I hope to use psychometric methods and statistical models to explore research questions related to fair and accurate psychological measurement. My main project focuses on quantifying the practical impact of item bias(noninvariance) on total scale scores. My secondary project evaluates the impact of two-level heteroscedasticity on small sample corrections for fixed-effects inferences. In addition, I am involved in projects on bootstrapping for multilevel models, multilevel regression and poststratification on subnational predictions with language data.
 
 - [CV](static/cv/YichiZhang_CV_20220206.pdf)
 - Email: yzhang97@usc.edu
+- Personal website: https://yichiyczhang.rbind.io/about.html 
```

Comment:
Should we use consistent formatting? So [Personal website](https://yichiyczhang.rbind.io/about.html)?